shell/artifacts/Social/Particles.manifest

```diff
@@ -6,6 +6,7 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
+import 'https://$cdn/artifacts/Common/List.manifest'
```

Comment:
The `$cdn` redirect is only necessary for Particles that don't live in `shell/artifacts`. You can use simpler local form if you like (e.g. `../Common/List.manifest`) .